test/controller/api/TestKafkaHealthCheck.scala

```diff
@@ -0,0 +1,104 @@
+package controller.api
+
+import controllers.KafkaManagerContext
+import controllers.api.KafkaHealthCheck
+import kafka.manager.utils.{CuratorAwareTest, KafkaServerInTest}
+import kafka.test.SeededBroker
+import play.api.Play
+import play.api.libs.json.Json
+import play.api.test.Helpers._
+import play.api.test.{FakeApplication, FakeRequest}
+import play.mvc.Http.Status.{BAD_REQUEST, OK}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.Try
+
+
+class TestKafkaHealthCheck extends CuratorAwareTest with KafkaServerInTest {
+  private[this] val broker = new SeededBroker("controller-api-test",4)
+  override val kafkaServerZkPath = broker.getZookeeperConnectionString
+  private[this] val duration = FiniteDuration(10,SECONDS)
+
+  private[this] val testClusterName = "kafka-hc-test-cluster"
+  private[this] val testTopicName = "kafka-hc-test-topic"
+
+  override protected def beforeAll() : Unit = {
+    super.beforeAll()
+    lazy val app : FakeApplication = {
+      FakeApplication()
```

Comment:
you need to pass in additionalConfiguration = Map("kafka-manager.zkhosts" -> kafkaServerZkPath) into FakeApplication.